packages/fxa-admin-server/src/gql/model/account.model.ts

```diff
@@ -46,6 +47,9 @@ export class Account {
   @Field((type) => [AttachedClient], { nullable: true })
   public attachedClients!: AttachedClient[];
 
+  @Field((type) => [MozSubscription], { nullable: true })
+  public subscriptions!: MozSubscription[];
```

Comment:
If preferred, I can change this to `mozSubscriptions` 🤷‍♀️

Reply:
Seems good as is.